templates/policies/sts_installer_permission_policy.json

```diff
@@ -177,9 +178,14 @@
                 "ec2:DescribeVpcEndpointServiceConfigurations",
                 "ec2:DescribeVpcEndpointServicePermissions",
                 "ec2:DescribeVpcEndpointServices",
-                "ec2:ModifyVpcEndpointServicePermissions"
+                "ec2:ModifyVpcEndpointServicePermissions",
+                "servicequotas:ListServices",
+                "servicequotas:GetRequestedServiceQuotaChange",
+                "servicequotas:GetServiceQuota",
+                "servicequotas:RequestServiceQuotaIncrease",
+                "servicequotas:ListServiceQuotas"
```

Comment:
@arendej, do we have in the roadmap to include these permissions?

My understanding is that we don't need them (even if they generate warnings) - instead we should be running preflights in the API using a different role.